src/paths.py

```diff
@@ -0,0 +1,11 @@
+HEALTHCHECK_PATH = "/healthcheck"
```

Comment:
I think the addition of this file reduces readability. Please can you undo this change.